herbert_core/utilities/classes/@object_lookup/private/parse_sz_repmat.m

```diff
@@ -0,0 +1,57 @@
+function sz_repmat_out = parse_sz_repmat (sz_repmat, n_objArr, ntot_objArr)
+% Check that the repmat option for implicitly expanding the stored arrays in an
+% object_lookup object is valid. Throw an error if not.
+%
+%   >> sz_repmat_out = parse_sz_repmat (sz_repmat, n_objArr, ntot_objArr)
+
+% Make sz_repeat a cell array for convenience
+if ~iscell(sz_repmat)
+    sz_repmat = {sz_repmat};    
+end
+
+% Check input is not empty
+nsz_repmat = numel(sz_repmat);
+if nsz_repmat==0
+    error('HERBERT:object_lookup:invalid_argument', ...
+        'If it has been given, the set of repeat sizes cannot be empty')
+end
```

Comment:
This doesn't do what you think. If `sz_repmat = []`, this will still respond positively because of `{sz_repmat}`